packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp

```diff
@@ -584,6 +580,7 @@ void ec_recover(MachineState& m) {
         assumeInt(m.stack[i / 4]) +=
```

Comment:
This looks like it still needs to be updated to return the address (hashed pubkey) instead of the pubkey.